razorpay/razorpay.php

```diff
@@ -1,44 +1,47 @@
 <?php
 
 require_once __DIR__.'/razorpay-sdk/Razorpay.php';
+require_once __DIR__.'/razorpay-webhook.php';
```

Comment:
why we need this

Reply:
@ramth05 instead of hardcoding the event name, considering it from the const defined in razorpay-webhook.php file.

Reply:
ok, let define the events in this file only. As we already including this file in webhook script.